k8s/testground-daemon/deployment.yml

```diff
@@ -20,20 +20,21 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       nodeSelector:
         testground.node.role.infra: "true"
-      initContainers:
-      - name: iproute-add
-        image: busybox:1.31.1
-        securityContext:
-          privileged: true
-        command:
-        - sh
-        - -ac
-        - >
-          while [ "$GW" = "" ]; do export GW=$(ip route | grep cni0 | awk '{print $7}'); echo "Got GW: $GW"; sleep 5; done;
-          echo $GW &&
-          ip route &&
-          ip route add 100.64.0.0/16 via $GW &&
-          ip route || true;
+      # Why this? it doesn't work
```

Comment:
At the time, this was necessary because there was no route added on Flannel to the Gateway, so we needed to add it automatically on Testground side. Namely:

`ip route add 100.64.0.0/16 via $GW`